packages/reporter/src/attempts/attempt-model.ts

```diff
@@ -180,7 +190,17 @@ export default class Attempt {
   _addSession (props: SessionProps) {
     const session = new Session(props)
 
-    this.sessions[props.sessionInfo.id] = session
+    this.sessions[props.id] = session
+  }
+
+  _updateSession (props: SessionProps) {
+    const session = this.sessions[props.id]
+
+    if (session) {
+      return session.update(props)
```

Comment:
wonder if it makes sense to return after the update for some reason the signature in `Session` changes, which also might imply that `_updateSession` is always a void return